quickwit/quickwit-metastore/src/metastore/file_backed_metastore/mod.rs

```diff
@@ -144,12 +144,12 @@ impl FileBackedMetastore {
     async fn mutate(
         &self,
         index_id: &str,
-        mutation: impl FnOnce(&mut FileBackedIndex) -> crate::MetastoreResult<bool>,
+        mutate_fn: impl FnOnce(&mut FileBackedIndex) -> crate::MetastoreResult<bool>,
     ) -> MetastoreResult<()> {
```

Comment:
I think we still need to propagate whether there was a mutation or not to the callers. `-> MetastoreResult<bool> `

Reply:
I'm not entirely sure what you intend to do with it, so I'll let you do it in your PR.